scripts/stats/cam/exevs_href_precip_stats.sh

```diff
@@ -65,7 +64,7 @@ fi
 
 # Build sub-jobs for snowfall
 if [ $verif_snowfall = yes ] ; then
- $USHevs/cam/evs_href_snowfall.sh
+ source $USHevs/cam/evs_href_snowfall.sh
```

Comment:
@MarcelCaron-NOAA @PerryShafran-NOAA Can you please explain to me why we are sourcing an /ush script here? 

CC @ShelleyMelchior-NOAA

Reply:
@AliciaBentley-NOAA This job would fail in data denial tests, when snowfall data (DCOMIN) are missing.  That ush file contains the check for the existence of snowfall data, and is the source of the issue.  With these changes, if snowfall data are missing, the ush file updates "verif_snowfall"--a boolean that determines whether or not to run "generate" and "gather" steps for snowfall--to "no".  Sourcing the script pulls the update into the ex-script.